README.MD

```diff
@@ -1,34 +1,51 @@
 ```
-.twowayset hhid tid
-.twowayset hhid tid w
+.varlist(numeric ts fv) [if] [in], [,ABSorb(varlist min=2 max=3) DROP GENerate(name)] [, NEWVars(name) REPLACE NOPROJ] [, VCE(name)]
 ```
-The first line uses assigns each observation the same weight. The second line assigns to each observation the weight given by variable w.
-The routine twowayset computes the first step of the algorithm and stores the results in the working directory. To store the results in a different folder or to avoid overwriting previous results use the option root. To store the results in files with a prefix abs in the folder C:\Results, write:
+In absorb option the user can define the two variables with the household id and the time id. From this option the user can choose two ways to deal with the redundant observations: drop them (typing drop after absorb) or generate a new variable that indicates which observations are not redundant (with gen() option).
+In absorb option the user must also specify the weight variable. 
+
+In the second list of options the user chooses how to perform the second step of the algorithm. The user has three options: create new variables that represent the residual projection of the variables in the list(in parenthesis the user choose the prefix of this new variables), replace variables with their projected counterpart or don't poject the variables if they were already projected.
+
+With vce() option the user chooses how to calculate the standar errors:
 ```
-.twowayset hhid tid w, root(“C:\Results\abc”)
+.vce(robust): standard errors robust to heteroscedasticity but assumes no correlation within group or serial correlation.
+.vce(vce):standard errors  proposed by Arellano (1987) robust to heteroscedasticity and serial correlation. 
+.vce(vce_2):Arellano standard errors with a degree of freedom correction performed by Stata xtreg, fe
+.If vce() is omitted: standard errors assuming homoscedasticity and no within  group correlation or serial correlation.
 ```
-projvar performs the second step of the algorithm. It is followed by a list of variables. These variables are projected onto the set of dummies, and the residual of this projection is saved as additional variables. Alternatively, the user may choose to replace the existing variables. For example:
+
+Here are some examples of the command:
 ```
-.projvar y x1 x2, p(w_)
+.twowayregwrap y x*, absorb(hhid tid w) drop newv(w_) vce(robust)
+.twowayregwrap w_y w_x1, absorb(hhid tid w) drop noproj vce(vce)
+In the first line the user chooses to drop the redundant variables and creates new variables that are the projected variables, whereas in the second line the user takes advantage of the variables already projected to make a faster regression.
 ```
-This command generates variables w_y, w_x1 and w_x2. Each variable contains the residual of the projection of the original variable on the set of dummies. 
+
+If the user wants to take more advantage of the algorithm it can be used separately.
```

Comment:
Replace with:

When running multiple specifications with the same dummy structure, the following commands can be used to avoid the repetition of the most time consuming steps.